CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# dbt_youtube_analytics v1.0.1
+
+[PR #24](https://github.com/fivetran/dbt_youtube_analytics/pull/24) includes the following updates:
+
+## Bug Fix
+- Updated the `get_video_columns` macro to use `dbt.type_timestamp()` instead of `datetime` datatype for the `snippet_published_at` and `status_publish_at` fields.
+ - This change corrects an error experienced by customers with Databricks SQL Warehouse destinations where `datetime` is not a supported datatype.
```

Comment:
It looks like we actually don't have the databricks-sql integration tests setup for this data model yet. Let's also include the relevant updates to ensure databricks-sql warehouse integration tests run for this model on future updates.